src/kv/RocksDBStore.cc

```diff
@@ -297,7 +297,8 @@ int RocksDBStore::do_open(ostream &out, bool create_if_missing)
 
   PerfCountersBuilder plb(g_ceph_context, "rocksdb", l_rocksdb_first, l_rocksdb_last);
   plb.add_u64_counter(l_rocksdb_gets, "rocksdb_get", "Gets");
-  plb.add_u64_counter(l_rocksdb_txns, "rocksdb_transaction", "Transactions");
+  plb.add_u64_counter(l_rocksdb_txns, "rocksdb_submit_transaction", "Submit transactions");
+  plb.add_u64_counter(l_rocksdb_txns_sync, "rocksdb_submit_transaction_sync", "Submit transactions sync");
```

Comment:
while you are here, do you mind removing the "rocksdb_" prefix from all of these?  they are already inside the "rocksdb" perfcounter section so it is redundant.